eslint.config.mjs

```diff
@@ -0,0 +1,47 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import path from 'node:path';
+import eslint from '@eslint/js';
+import { includeIgnoreFile } from '@eslint/compat';
+import tseslint from 'typescript-eslint';
+import eslintPrettier from 'eslint-plugin-prettier/recommended';
+import unicornPlugin from 'eslint-plugin-unicorn';
+import headerPlugin from 'eslint-plugin-header';
+
+// https://github.com/Stuk/eslint-plugin-header/issues/57
+headerPlugin.rules.header.meta.schema = false;
+
+export default tseslint.config(
+  includeIgnoreFile(path.resolve('.gitignore')),
+  eslint.configs.recommended,
+  tseslint.configs.recommended,
+  eslintPrettier,
+  {
+    files: ['**/*.{js,mjs,ts}'],
+    languageOptions: {
+      globals: {
+        console: true,
```

Comment:
New syntax for globals. Instead of `"browser": true` preset, they now require listing all constants one-by-one. They recommend [pulling an extra dependency](https://www.npmjs.com/package/globals) for a reusable list of globals, but since we only have one, I decided to keep it simple